web/src/cypress/integration/admin.ts

```diff
@@ -48,7 +48,7 @@ function testAdmin(screen: ScreenFormat) {
       cy.get('button[type="submit"]')
         .contains('Confirm')
         .click()
-      cy.get('div[role="document"]').should('not.exist') // dialog should close
+      cy.get('div[role="document"]').should('exist') // dialog should close
```

Comment:
Were these tests failing for you? I believe the div with `role="document"` is the dialog, and shouldn't exist anymore after Cypress clicks on Confirm

Reply:
I see the problem, I'll fix that.